docs/extend/modular-resources/examples/custom-arm.md

```diff
@@ -0,0 +1,258 @@
+tags: ["motor", "odrive", "canbus", "serial", "module", "modular resources", "Python", "python SDK", "CAN"]
+# SMEs: Nicole Jung
+---
+
+The RDK provides a number of built-in {{< glossary_tooltip term_id="model" text="models" >}} that implement the API protocol of the [arm](/components/arm/) subtype of component, such as the `ur5e`, `xArm6`, and `xArm7`.
+
+{{% alert title="Info" color="info" %}}
+
+*Built-in* indicates each of these models has a driver for their native software in the RDK.
+For example, the `ur5e`'s driver is defined in the RDK as found on [GitHub](https://github.com/viamrobotics/rdk/blob/main/components/arm/universalrobots/ur.go).
+
+Each of these models also has to have a [kinematics file](/internals/kinematic-chain-config/) that specifies the relative [orientation](/internals/orientation-vector/) of components in its kinematic chain, which is located in the RDK in the same directory as the respective driver.
+For example, the `ur5e`'s kinematics file, <file>ur5e.json</file>, is defined in the RDK as found on [GitHub](https://github.com/viamrobotics/rdk/blob/main/components/arm/universalrobots/ur5e.json).
+
+See [Arm Configuration](/components/arm/#configuration) for the current list of built-in models the RDK provides.
+
+{{% /alert %}}
+
+If you have a robot arm whose native software is not provided with a built-in driver, you probably don't want to replace that arm **($$$)**, but you might want to use the [Arm API](/components/arm/#api) to [program](/program/) and control your arm, or want to use it with a Viam [service](/services/).
+
+Follow these instructions to implement a model of [arm component](/components/arm/) that is not built-in to the RDK:
+
+- [Get your arm's kinematics file](#get-your-arms-kinematics-file)
+- [Create a custom arm model as a modular resource](#create-a-custom-arm-model-as-a-modular-resource)
+  - [Code a new resource model](#code-a-new-resource-model)
+  - [Code a main entry point program](#code-a-main-entry-point-program)
+  - [Compile the module into an executable](#compile-the-module-into-an-executable)
+- [Configure the module and modular resource on your robot](#configure-the-module-and-modular-resource-on-your-robot)
+
+This guide uses Viam's Python SDK to code the arm module, but if you want to use the Go Client SDK, you can.
+Follow [this guide](/extend/modular-resources/create/#code-a-new-resource-model) and select **Go** to learn how to code a module providing a new arm model in Go.
+
+{{% alert title="Modules vs. modular resources" color="tip" %}}
+
+The Viam module system allows you to integrate custom {{< glossary_tooltip term_id="resource" text="resources" >}} ([components](/components/) and [services](/services/)) into any robot running on Viam.
+
+A configured *module* can make one or more *modular resources* available for configuration.
+
+{{% /alert %}}
+
```

Comment:
```suggestion
```
I think this clutters the page and is sufficiently covered in the other pages. I think if people don't understand this, they should go to the modular resources concepts page etc.